### ISSUE TITLE
panic: dive error! can't dive on a non slice or map

### ISSUE THREAD

**Issue by netixx** [akvorado/akvorado#993]
# Bug description

Orchestrator fails to parse static metadata provider config.
I also tried with the configuration from the docs: https://demo.akvorado.net/docs/configuration#metadata, and had the same results (I double-checked, but I hope my copy-pasting skills were not the issue).

## Steps to reproduce the problem

**Akvorado version**: 
```
akvorado v1.9.2
  Built with: go1.21.1
```

configure static provider:
```
metadata:
  workers: 10
  provider:
    type: static
    exporters:
      "0.0.0.0/0":
        name: default
        default:
          name: unknown
          description: Unknown
          speed: 100
        ifindexes: 
          0:
            name: none
            description: none
            speed: 1000
```

For reference, I am trying to setup a dummy provider to get the flows inside the pipeline, so that we can have a baseline, and continue enriching after that.

## Expected outcome
it works :) 
## Current outcome

Panic on orchestrator start:
```
akvorado-akvorado-orchestrator-1  | panic: dive error! can't dive on a non slice or map
akvorado-akvorado-orchestrator-1  | 
akvorado-akvorado-orchestrator-1  | goroutine 1 [running]:
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1c195e0?, 0xc000627cc0?, 0x4f9b8a?}, {0x1c19520?, 0xc000627cd0?, 0xc0008be0c8?}, {0xc000e00180, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:317 +0x298c
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1bba740?, 0xc00018d9a0?, 0x1aa25c0?}, {0x1c195e0?, 0xc000627cc0?, 0x7f65cfa7c108?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1bba740?, 0xc00018d9a0?, 0x0?}, {0x1c195e0?, 0xc000627cc0?, 0x1?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:197 +0x19f7
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1bba740?, 0xc00018d9a0?, 0x5bffffffffffffff?}, {0x1bfaf40?, 0xc00018d9a0?, 0x50000c0006733a8?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:310 +0x2331
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1beaaa0?, 0xc000879670?, 0x0?}, {0x1bba740?, 0xc00018d9a0?, 0x1?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1beaaa0?, 0xc000879670?, 0x1a9ea3e?}, {0x1b67240?, 0xc000879670?, 0x8ecb9f?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:197 +0x19f7
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1cc01c0?, 0xc000879648?, 0x40?}, {0x1beaaa0?, 0xc000879670?, 0x1?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1cc01c0?, 0xc000879648?, 0xeffffffffffffff?}, {0x1beaaa0?, 0xc000879670?, 0x70000c0006720a0?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:197 +0x19f7
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1cdefe0?, 0xc000879600?, 0x220?}, {0x1cc01c0?, 0xc000879648?, 0x1?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1cdefe0?, 0xc000879600?, 0x0?}, {0x1cc01c0?, 0xc000879648?, 0x1?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:197 +0x19f7
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1ceff80?, 0xc00043b880?, 0xc0008bf090?}, {0x1cdefe0?, 0xc000879600?, 0xc000b2de40?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1ceff80?, 0xc00043b880?, 0xc00043b880?}, {0x1cdefe0?, 0xc000879600?, 0xc00043bb38?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:197 +0x19f7
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).traverseField(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1ceff80?, 0xc00043b880?, 0x110?}, {0x1a7e5a0?, 0xc00043bbb8?, 0x7f6588a692d0?}, {0xc000b2de40, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:273 +0x242e
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*validate).validateStruct(0xc00042bb00, {0x272a4c8, 0x35e1320}, {0x1b19200?, 0xc00043b880?, 0xc0008bf6f0?}, {0x1ceff80?, 0xc00043b880?, 0x4103a5?}, {0x274e818, ...}, ...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator.go:77 +0x745
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*Validate).StructCtx(0xc000570380, {0x272a4c8, 0x35e1320}, {0x1b19200?, 0xc00043b880?})
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator_instance.go:393 +0x445
akvorado-akvorado-orchestrator-1  | github.com/go-playground/validator/v10.(*Validate).Struct(...)
akvorado-akvorado-orchestrator-1  |     github.com/go-playground/validator/v10@v10.15.5/validator_instance.go:366
akvorado-akvorado-orchestrator-1  | akvorado/cmd.ConfigRelatedOptions.Parse({{0x7ffc4788ff1f?, 0x0?}, 0x0?, 0xc000699850?}, {0x271e000, 0xc00018c030}, {0x1da1361, 0xc}, {0x1b19200, 0xc00043b880})
akvorado-akvorado-orchestrator-1  |     akvorado/cmd/config.go:149 +0xe73
akvorado-akvorado-orchestrator-1  | akvorado/cmd.glob..func6(0xc00062ac00?, {0xc000699820, 0x1, 0x1d698a4?})
akvorado-akvorado-orchestrator-1  |     akvorado/cmd/orchestrator.go:88 +0x165
akvorado-akvorado-orchestrator-1  | github.com/spf13/cobra.(*Command).execute(0x35920c0, {0xc0006997f0, 0x1, 0x1})
akvorado-akvorado-orchestrator-1  |     github.com/spf13/cobra@v1.7.0/command.go:940 +0x87c
akvorado-akvorado-orchestrator-1  | github.com/spf13/cobra.(*Command).ExecuteC(0x3591260)
akvorado-akvorado-orchestrator-1  |     github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3a5
akvorado-akvorado-orchestrator-1  | github.com/spf13/cobra.(*Command).Execute(0x0?)
```

Not sure what the root cause here is, but maybe SubnetMap is not a map ?
```
// in inlet/metadata/provider/static/config.go
	Exporters *helpers.SubnetMap[ExporterConfiguration] `validate:"omitempty,dive"`
```

**Comment by vincentbernat** [comment]
This is fixed in 349003307d4b981309b08b7010aa26a1d941edff (not released).

**Comment by netixx** [comment]
ah cool thanks!
